webrtc.html

```diff
@@ -203,9 +233,13 @@
 
         <p>An example array of RTCIceServer objects is:</p>
 
-        <p><code>[ { "urls": "stun:stun1.example.net" }, { "urls":
-        "turn:turn.example.org", "username": "user", "credential": "myPassword"
-        } ]</code></p>
+        <p><code>[
+          { "urls": "stun:stun1.example.net" },
+          { "urls": "turn:turn.example.org",
+            "username": "user",
```

Comment:
and maybe an example of urls plural while we're at it (preferably with turn where username and password are reused)?